Dockerfile-alpine

```diff
@@ -0,0 +1,39 @@
+# Usage:
```

Comment:
Do we need these docker files?

The current CI simply uses Github actions inside docker

Reply:
They are useful if you want to do xplatform test/ build locally without relying on a GH workflow.  

Otherwise you can't test or build for those targets unless you create a PR and wait for the actions to complete.  It's noticeably slower than running the Docker builds locally as well.

Reply:
Oh actually they are also used by the build-pack-publish github workflow.  The existing ci workflow does a test on libc and alpine but not for arm/ arm64.

Looking at it now, *maybe* it is possible to do a matrix of jobs with `container.image` like `[ 'arm64v8/node:14-bullseye', 'arm32v7/node:14-bullseye', 'node:14-bullseye']` and also one for alpine, then collect all of the build artifacts for both in the 'pack' step of that workflow.

Reply:
I guess you can also remove 'test-alpine.sh' since that is not using prebuildify and is covered in the docker build.

Reply:
My concern is mostly around duplicating the Dockerfile logic across all combinations of architecture we plan to support.

One thing we can do here is to have 2 Dockerfiles (one for Debian, one for Alpine) and pass the actual container image as a Dockefile `ARG`

My current way of building Alpine ARM builds is running `test-alpine.sh` under an arm64 docker container.

Reply:
I think you are referring to [this](https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact) I've never tried that before, let me give it a try.